tests/functional/features/chart_submitted_by_unauthorized_user.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Chart submitted by unauthorized user
+    Partners or redhat associates can not submit charts if they are not in the OWNERS file
+
+    Scenario: A unauthorized user submits a chart with report 
+        Given user is not present in the OWNERS file of the chart
```

Comment:
Indentation should be the same for Given, When, Then and And.

Reply:
Specify what user either in the Scenario or Given: Partners / redhat associates

Reply:
Indentation changes added.